core/store/src/lib.rs

```diff
@@ -46,7 +46,7 @@ impl Store {
         self.storage.get(column, key).map_err(|e| e.into())
     }
 
-    pub fn get_ser<T: BorshDeserialize>(
+    pub fn get_ser<T: BorshDeserialize + Clone>(
```

Comment:
Why is `Clone` needed here?

Reply:
Fixed. Thanks for the catch, that was left from an experimental iteration.